### PR TITLE
Add cron schedule and coverage upload to coveralls

### PR DESCRIPTION
**Pull request by marcushines**

### CODE REVIEW

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -5,59 +5,55 @@ on:
...
+      - name: Set up Go
+        uses: actions/setup-go@v2.1.3
+        with:
+          go-version: '1.18'
+      - name: Run Coverage
+        run: go test -v -coverprofile=profile.cov ./...
+      - name: Send coverage
+        uses: shogo82148/actions-goveralls@v1
+        with:
+          path-to-profile: profile.cov
```
Comment (robshakir) [inline .github/workflows/go.yml:37]:
```suggestion
          path-to-profile: profile.cov
          
```

File: .github/workflows/go.yml [hunk 2]
```diff
@@ -5,59 +5,55 @@ on:
...
     name: Static Analysis
     runs-on: ubuntu-latest
-
     steps:
+      - uses: actions/checkout@v2
       - name: Install Go
         uses: actions/setup-go@v2
         with:
-          go-version: '1.17'
+          go-version: '1.18'
```
Comment (robshakir) [inline .github/workflows/go.yml:46]: Just `1.x` here perhaps?

**Review by robshakir — APPROVED** [review]
Couple of minor suggestions.

File: .github/workflows/go.yml (hunk 2 above)
Comment (marcushines) [reply .github/workflows/go.yml:47]: ack

**Pull request merged**